Sources/Sentry/SentryBreadcrumbTracker.m

```diff
@@ -134,12 +134,35 @@ - (void)swizzleSendAction
     SentrySwizzleInstanceMethod(UIApplication.class, selector, SentrySWReturnType(BOOL),
         SentrySWArguments(SEL action, id target, id sender, UIEvent * event), SentrySWReplacement({
             if (nil != [SentrySDK.currentHub getClient]) {
-                NSDictionary *data = nil;
+                NSMutableDictionary *data = nil;
                 for (UITouch *touch in event.allTouches) {
                     if (touch.phase == UITouchPhaseCancelled || touch.phase == UITouchPhaseEnded) {
-                        data = @ { @"view" : [NSString stringWithFormat:@"%@", touch.view] };
+                        data = @ { @"view" : [NSString stringWithFormat:@"%@", touch.view] }
+                                   .mutableCopy;
+
+                        if ([sender isKindOfClass:UIView.class]) {
```

Comment:
`m`: Can we add some tests for this new logic, please? The easiest way to make this testable would be to add an extra function for this logic and just call it directly from the tests and validate the data, for example.